### DISCUSSION TITLE
Did you use Remix in a commercial project?

### DISCUSSION THREAD

**Discussion by JSsantana** [remix-run/remix#2770] [General]
I'm thinking about use Remix into a product of the company I work for, but I saw some articles saying that is not a good choice for "commercial projects" only for study and personal projects, because it is "to recent". 

Do you have any experience of using Remix in a commercial product? What is your opinion?

**Comment by sergiodxa** [comment]
I’m using Remix at work in https://daffy.org, we are migrating away from Next, some URLs are still in Next, some are in Remix, all Remix code is way simpler.

**Comment by blasterbug** [comment]
My company is transitioning their main product from React + Node.js backend to Remix.
